lambda_fns/extract_docs/app.py

```diff
@@ -68,6 +76,39 @@ def store_text_s3(data, bucket_name, key):
         Key=key
     )
 
+def upload_file_to_s3(
+    url,
+    key="temporaryfile.pdf",
+    bucketname="deep-large-docs-conversion"
+):
+    try:
```

Comment:
Maybe define a class/object for handling upload/download/URL.

Something like this:
Definition: https://github.com/the-deep/serverless/blob/develop/src/common/s3.py
Usages: https://github.com/the-deep/serverless/blob/develop/src/functions/source_extract/models.py#L92-L109

Reply:
done